src/Analyzer.Cli.FunctionalTests/ValidTemplateConstants.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Analyzer.Cli.FunctionalTests
+{   /// <summary>
+/// Constants being passed in as test cases, first three contants should pass while the following five should fail.
+/// </summary>
```

Comment:
```suggestion
{
    /// <summary>
    /// Constants being passed in as test cases, first three contants should pass while the following five should fail.
    /// </summary>
```

Reply:
fixing the spacing

Reply:
I can imagine in the future the order of the file will change. 
Instead, put the summary as "Constants being passed in as test cases for the CommandLineParserTests"
And on-top of each string, add if it will pass or fail.

Reply:
also type on "contants"

Reply:
Or a last option would be to add Pass/Fail at the beginning of the string variable